srcpkgs/sbs/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'sbs'
+pkgname=sbs
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="imlib2-devel libXinerama-devel"
+depends="imlib2-devel libXinerama-devel"
+short_desc="single purpose non-bloat background setter"
+maintainer="Onur Ozkan <work@onurozkan.dev>"
+license="GPL-2.0-only"
+homepage="https://github.com/ozkanonur/sbs"
+changelog="https://raw.githubusercontent.com/ozkanonur/sbs/master/ChangeLog"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=aa4da5b2315046fc2059599b19c530f08bb870e63ed17111a55991b1ae911367
+
+do_install() {
+	vbin sbs
+}
```

Comment:
why not let the default do_install do it? it should run `make install` just fine